tests/e2e/README.md

```diff
@@ -115,6 +133,7 @@ be a SHA which you need to copy and set it as a GIT_SHA. Example from a log: the
 
 * Then set the export variables again
 ```
+export LOCALREG=false
```

Comment:
isn't this default anyway?

Reply:
If the user makes use of a remote registry(e.g. docker hub, GCR), then local registry should be disabled.